components/Search.tsx

```diff
@@ -0,0 +1,141 @@
+import { View, StyleSheet, Text, ScrollView } from "react-native"
+import Input from "@/components/ui/Input"
+import UserInfo from "@/components/UserInfo"
+import CheckBox from "@/components/ui/CheckBox" // You can remove this if not using
+import ShareCard from "@/components/Share"
+import { CustomButton } from "@/components/ui/Button"
+import { useState } from "react"
+import { AntDesign } from "@expo/vector-icons"
+import { availableFriends } from "@/data/users"
+import { useNavigation } from "@react-navigation/native"
+import UserAvatar from "@/components/ui/UserAvatar"
+
+const FindFriends = () => {
+  const navigation = useNavigation()
+  const [search, setSearch] = useState("")
+  const [addedFriends, setAddedFriends] = useState<{ [key: string]: boolean }>(
+    {},
+  )
```

Comment:
isn't the `addedFriends`supposed to be an array of users?
```suggestion
  const [addedFriends, setAddedFriends] = useState<User[]>([])  )
```